analyses/stackanalyses/controller.go

```diff
@@ -190,7 +193,64 @@ func (mc *Controller) buildFileStats(manifestFile string) *driver.ReadManifestRe
 func (mc *Controller) getManifestName(manifestFile string) string {
 	stats, err := os.Stat(manifestFile)
 	if err != nil {
-		log.Fatal().Err(err).Msgf("Error")
+		log.Fatal().Err(err).Msgf("Error fetching manifest name.")
 	}
 	return stats.Name()
 }
+
+// ProcessResult processes result in CI Required format, return true if Vul found
+func ProcessResult(analysedResult driver.GetResponseType) bool {
+	totalDepsScanned := len(analysedResult.AnalysedDeps)
+	publicVulCount, privateVulCount, severities := processVulnerabilities(analysedResult.AnalysedDeps)
+	out := &driver.CIFormat{
+		TotalScannedDependencies:                   totalDepsScanned,
+		DirectDependenciesWithKnownVulnerabilities: publicVulCount,
+		DirectDependenciesWithSynkAdvisories:       privateVulCount,
+		VulnerabilitiesPerSeverity:                 *severities,
+	}
+	b, err := json.Marshal(out)
+	if err != nil {
+		log.Fatal().Msg("Error forming CLI Response.")
+	}
+	fmt.Fprintln(os.Stdout, string(b))
+	if out.DirectDependenciesWithKnownVulnerabilities+out.DirectDependenciesWithSynkAdvisories > 1 {
+		// If Vulnerability found, return true
+		return true
+	}
```

Comment:
Don't combine this new logic into golang PR, this can go separately as it needs a discussion.

Reply:
FYI, I did had a discussing, for other client integrations (VSCode etc) we are coming with `flag` which will totally skip `ProcessResult`  execution. 
I think we can go forward with EPIC requirements as of now. I will add the respective `flag` in the next PR.

Reply:
I'm still in the opinion of moving non golang related changes out of this PR.

Reply:
Ack.